app/src/main/java/com/soen490chrysalis/papilio/viewModel/factories/CreateActivityViewModelFactory.kt

```diff
@@ -17,9 +19,12 @@ class CreateActivityViewModelFactory : ViewModelProvider.NewInstanceFactory()
     {
         // Initialize Firebase Auth and inject it into the user repository
         val firebaseAuth = FirebaseAuth.getInstance()
+        //performance testing for retrofit service
+        val myTrace = Firebase.performance.newTrace("activity_api_retrofit_service")
```

Comment:
Same thing here, use a better variable name instead of "myTrace".
Also, I would use another name for the trace like _retrofit-service-boot-time_